doc/source/API/DesktopMessenger.rst

```diff
@@ -1,7 +1,7 @@
-Desktop and Messenger
```

Comment:
It might be useful to rename this file to `Messenger.rst`.

Reply:
@jleonatti Agreed--but I wasn't sure if they had additional plans to add to it. If nothing gets added, I'll rename it.